tests/base/dataloaders.py

```diff
@@ -21,3 +21,29 @@ def __next__(self):
         except StopIteration:
             self.iter = iter(self.dataloader)
             return next(self.iter)
+
+
+class CustomNotImplementedErrorDataloader:
+
+    def __init__(self, dataloader):
+        self.dataloader = dataloader
+        self.iter = iter(dataloader)
+        self.count = 0
+
+    def __len__(self):
+        """raise NotImplementedError"""
+        raise NotImplementedError
```

Comment:
this shall be rather missing...

Reply:
This is needed for testing the bug-fix dealing with a 'NotImplementedError' raised by a data loader. The fix was caused by torchtext which has __len__ defined in their data loader but raises 'NotImplementedError' in some circumstances if it can not compute the number of batches. E.g. in my case because the batch-size is determined dynamically.